docs/Installation-iOS.md

```diff
@@ -16,11 +16,12 @@ Unless otherwise stated, all of the commands shown in the following instructions
 	* iOS 12.0 or later:
 		* [Procursus](https://apt.procurs.us/) or [Bingner/Elucubratus](https://apt.bingner.com/) (this depends on the jailbreak you are using)
 	* Prior to iOS 12.0:
-		* [Coolstar's repo](https://coolstar.org/publicrepo/), [Sam Bingner's repo](https://repo.bingner.com/), and [BigBoss](http://apt.thebigboss.org/repofiles/cydia/)
+		* [Sam Bingner's repo](https://repo.bingner.com/) and [BigBoss](http://apt.thebigboss.org/repofiles/cydia/)
+		* Also check to ensure the non-(lib) (i.e., commandline tool) of APT is installed (e.g., APT 0.7 Strict on iOS 8)
```

Comment:
Looks great! Since `apt` is provided by https://www.ios-repo-updates.com/repository/cydia-telesphoreo/package/apt7/ on older builds, I think this may be better generalized:
```suggestion
		* Also check to ensure `apt` (i.e., the command-line tool of APT) is installed 
```